integral_view/views/monitoring.py

```diff
@@ -146,30 +137,154 @@ def view_remote_monitoring_server_status(request):
+        if err:
+            raise Exception(err)
+        if 'event_type_id' not in req_params or int(req_params['event_type_id']) not in reference_table_entries['reference_event_types'].keys():
+            raise Exception('Invalid request. Please use the menus.')
+        return_dict['event_type_id'] = req_params['event_type_id']
+        event_type_id = int(req_params['event_type_id'])
+        if request.method == "GET":
+            if event_type_id == 1: 
+                form = monitoring_forms.AlertNotificationsForm(reference_subsystem_types = reference_table_entries['reference_subsystem_types'], reference_severity_types = reference_table_entries['reference_severity_types'], reference_notification_types = reference_table_entries['reference_notification_types'])
+                template = 'create_alert_notification.html'
+            elif event_type_id == 2: 
+                form = monitoring_forms.AuditNotificationsForm(reference_notification_types = reference_table_entries['reference_notification_types'])
+                template = 'create_audit_notification.html'
+            return_dict['form'] = form
+            return django.shortcuts.render_to_response(template, return_dict, context_instance=django.template.context.RequestContext(request))
+
+        elif request.method == "POST":
+            scheduler = req_params['scheduler']
+            schedule = scheduler.split()
+            if event_type_id == 1:
+                form = monitoring_forms.AlertNotificationsForm(request.POST, reference_subsystem_types = reference_table_entries['reference_subsystem_types'], reference_severity_types = reference_table_entries['reference_severity_types'], reference_notification_types = reference_table_entries['reference_notification_types'])
+                template = 'create_alert_notification.html'
+            elif event_type_id == 2: 
+                form = monitoring_forms.AuditNotificationsForm(request.POST, reference_notification_types = reference_table_entries['reference_notification_types'])
+                template = 'create_audit_notification.html'
+            return_dict['form'] = form
+            if not form.is_valid():
+                return django.shortcuts.render_to_response(template, return_dict, context_instance=django.template.context.RequestContext(request))
+            cd = form.cleaned_data
+
+            psp, err = config.get_python_scripts_path()
+            if err:
+                raise Exception(err)
+            if 'subsystem_type_id' in cd:
+                subsystem_type_id = int(cd['subsystem_type_id'])
+            else:
+                subsystem_type_id = -1
+            if 'severity_type_id' in cd:
+                severity_type_id = int(cd['severity_type_id'])
+            else:
+                severity_type_id = -1
```

Comment:
When `event_type_id` is 2, `severity_type_id` being `-1`(which is 'All' type) is not of any relevance, is it? If we default the value to 'None' in the table schema, we need not worry about it at all.